projects/aca-shared/src/lib/components/tool-bar/toolbar-menu/toolbar-menu.component.html

```diff
@@ -12,14 +12,19 @@
 </button>
 
 <mat-menu #menu="matMenu" [overlapTrigger]="false">
-  <ng-container *ngFor="let child of actionRef.children; trackBy: trackByActionId">
-    <ng-container [ngSwitch]="child.type">
```

Comment:
is there a reason for removing this container?

Reply:
No there isn't a reason. Its likely that this was removed while trying out different approaches, and then slipped through during the code push. I'll revert this change in the next commit.